server/pbench/bin/pbench-sync-package-tarballs

```diff
@@ -83,24 +84,9 @@ if [[ $status != 0 ]] ;then
     exit 2
 fi
 
-typeset -i nerrs=0
-typeset -i nprocessed=0
-
-nprocessed=$(wc -l < $tmp/files-to-sync)
-if [ -s $tmp/mail.log ]; then
-    nerrs=$(wc -l < $tmp/mail.log)
-fi
-
-# if there are errors, send mail
-if [ "$nerrs" -gt 0 ]  ;then
-    index_content=$tmp/index_mail_contents
-    cat << EOF > $index_content
-$PROG: Processed $nprocessed result tar balls, with $nerrs errors
+log_finish
 
-EOF
-    cat $tmp/mail.log >> $index_content
```

Comment:
Since we are no longer collecting from `$tmp/mail.log`, perhaps we should change the other references to `mail.log` to `>&4` instead?